server/src/main/java/org/elasticsearch/index/fielddata/plain/SortedDoublesIndexFieldData.java

```diff
@@ -43,17 +43,28 @@ public class SortedDoublesIndexFieldData extends IndexNumericFieldData {
     public static class Builder implements IndexFieldData.Builder {
         private final String name;
         private final NumericType numericType;
+        private final ValuesSourceType valuesSourceType;
         protected final ToScriptFieldFactory<SortedNumericDoubleValues> toScriptFieldFactory;
 
         public Builder(String name, NumericType numericType, ToScriptFieldFactory<SortedNumericDoubleValues> toScriptFieldFactory) {
```

Comment:
Do we need the old ctor? Does `NumericType` need a `ValuesSourceType` any more?

Reply:
We can nuke the old constructor.  I think `NumericType` still needs a `ValuesSourceType`, or rather than removing it will be a much bigger change.

Reply:
It can wait, that's cool